### DISCUSSION TITLE
[Feature Request] Color Picker

### DISCUSSION THREAD

**Discussion by gokhangunduz** [heroui-inc/heroui#4740] [Feature requests]
### Is your feature request related to a problem? Please describe.

Hello everyone. I have been using this library for a long time. I love this. But i need color-picker at some project. I think the process of creating the ColorPicker component is conceivable.

### Describe the solution you'd like

New Component

### Describe alternatives you've considered

None

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1086/[feature-request]-color-picker">ENG-1086 [Feature Request] Color Picker</a></p>

**Comment by awesome-pro** [comment]
Can your give some idea/reference/example what sort of color picker  you want @gokhangunduz

**Comment by gokhangunduz** [comment]
I think the best reference;

https://primereact.org/colorpicker/

**Comment by gokhangunduz** [comment]
Hi, any progress ? I'm looking forward to. Thanks @abhinandan-verma

**Comment by awesome-pro** [comment]
@gokhangunduz  currently the team is busy with zigma,
And I need their acceptance for any new component, i will inform youe soon, once the team gives acceptance for the component:)

**Comment by cyberxnomad** [comment]
any progress?

**Reply by cyberxnomad** [reply]
Sorry, I was reading the v2 documentation. Heroui v3 already has a color picker.
